### DISCUSSION TITLE
duan-zi/info/oLQe4O-htR1scZHisgTmJ

### DISCUSSION THREAD

**Discussion by giscus** [wurencaideli/dumogu-ypwz-comment#25] [Announcements]
# duan-zi/info/oLQe4O-htR1scZHisgTmJ

不是只有视频和音乐才带给人欢乐！还有一篇文字哟！学生时代所看过的短文还有谁记得呢？甚是怀念呢！

https://www.dumogu.top/duan-zi/info/oLQe4O-htR1scZHisgTmJ?isInfo=false

**Comment by luo347070642** [comment]
超支了，小月每日可支配零花钱绝对超过 100
